index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Harshika's Portfolio</title>
+    <link rel="stylesheet" href="style.css" class="css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://kit.fontawesome.com/3ab8daaa6e.css" crossorigin="anonymous">
+    <script src="https://kit.fontawesome.com/3ab8daaa6e.js" crossorigin="anonymous"defer></script>
+</head>
+<body>
+    <header>
+
+    <nav class="mobileinfo">
+        <a class="mobilelogo"><h1>Welcome</h1></a>
+          </div>
+          
+          <div class="mobilemenu">
+          <i class="fa-sharp fa-solid fa-bars"></i>  
+           </div>
+          </nav>
+          
+          
+        <nav class="desktop-nav">
+            <ul class="desktop-navmenu">
+                <li><a href ="#Portfolio">Portfolio</a></li>
+                <li><a href="#About">About</a></li>
+                <li><a href="#Contact">Contact</a></li>
+
+            </ul>
+        </nav>
```

Comment:
- [Optional] Great job for adding desktop navigation also :+1: I think it would be better to use one `nav` for mobile hamburger and desktop navigation :+1: